qa/tasks/ceph.py

```diff
@@ -653,7 +653,7 @@ def cluster(ctx, config):
            (remote.ssh.get_transport().getpeername() for (remote, role_list) in remotes_and_roles)]
     mons = get_mons(
         roles, ips, cluster_name,
-        mon_bind_msgr2=config.get('mon_bind_msgr2'),
+        ms_bind_msgr2=config.get('ms_bind_msgr2'),
         mon_bind_addrvec=config.get('mon_bind_addrvec'),
```

Comment:
There does not seem to be any ceph.conf option called `mon_bind_addrvec` (or even `ms_bind_addrvec` for that matter) so I wonder if setting `mon_bind_addrvec` actually does anything?